ksqldb-functional-tests/src/test/resources/query-validation-tests/kafka.json

```diff
@@ -7,7 +7,7 @@
         "CREATE STREAM INPUT (K STRING KEY, foo INT) WITH (WRAP_SINGLE_VALUE=true, kafka_topic='input', value_format='KAFKA');"
       ],
       "expectedException": {
-        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "type": "io.confluent.ksql.util.KsqlException",
```

Comment:
Can the `KsqlStatementException` stay? There is a check at the end of the `KsqlResource.handleKsqlStatements()` method that returns a different value depending on the exception.

This question applies for the rest of the tests that changes to this new exception.

Reply:
Sure. Will catch `KsqlException` and rethrow